rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] As the `attr_accessor` creates both the READER & WRITER methods. I would recommend including the `book and person` arguments in the attr_accessor too.

Reply:
Thanks for the feeback 
i have resolved it